### ISSUE TITLE
[BUG] possible bug in Rollout Baseline

### ISSUE THREAD

**Issue by fedebotu** [ai4co/rl4co#39]
In both mTSP and PDP, with rollout baseline, we may get an exploding behavior (loss increases after some time)
I suspect this may be due to gradient clipping by PyTorch Lightning, so we may have to investigate
